source/render/MovableTextOverlay.cpp

```diff
@@ -137,6 +137,14 @@ void ChildOverlay::update(Ogre::Real timeSincelastFrame)
     mOverlayContainer->hide();
 }
 
+void ChildOverlay::isOnScreen(bool onScreen)
```

Comment:
Maybe ``setOnScreen`` would be a better name? We tend to use ``isSomething`` names for methods that return a boolean.